[Digital Twins - Core] Build test package failing in nightly runs

Digital Twins - Core Share nightly test runs are failing with:

> Error message:
> Has no exported member named 'isRestError'. Did you mean 'RestError'?
> 
> Stack trace:
>Found configuration in /mnt/vss/_work/1/s/rush.json
Rush Multi-Project Build Tool 5.83.4 - Node.js 16.18.1 (LTS)
 "tsc -p ."
testComponents.spec.ts(8,10): error TS2724: '"@azure/core-rest-pipeline"' has no exported member named 'isRestError'. Did you mean 'RestError'?
testDigitalTwins.spec.ts(14,10): error TS2724: '"@azure/core-rest-pipeline"' has no exported member named 'isRestError'. Did you mean 'RestError'?
testModels.spec.ts(9,10): error TS2724: '"@azure/core-rest-pipeline"' has no exported member named 'isRestError'. Did you mean 'RestError'?
testRelationships.spec.ts(8,10): error TS2724: '"@azure/core-rest-pipeline"' has no exported member named 'isRestError'. Did you mean 'RestError'?
The script failed with exit code 2


For more details check here:

- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2111791&view=results

--- a/sdk/digitaltwins/digital-twins-core/package.json
+++ b/sdk/digitaltwins/digital-twins-core/package.json
@@ -66,7 +66,7 @@
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
     "@azure/core-client": "^1.7.1",
-    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/core-rest-pipeline": "^1.8.1",
     "@azure/core-paging": "^1.1.1",
     "@azure/core-tracing": "^1.0.0",
     "@azure/core-util": "^1.0.0",

